docs/pom.xml

```diff
@@ -297,11 +297,24 @@
                 </executions>
             </plugin>
             <plugin>
-                <!--
-                    Override the javadoc configuration to include source jars from our dependencies in order to build combined
-                    documentation for the entire distribution.
-                -->
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <goals>
+                            <goal>unpack-dependencies</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <classifier>sources</classifier>
+                            <includeGroupIds>gov.nsa.datawave, gov.nsa.datawave.webservices</includeGroupIds>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
                 <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.0.1</version>
```

Comment:
Just curious why this version isn’t coming from the parent pom?

Reply:
removed